src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -211,6 +211,27 @@ def __init__(self, app, config, mount):
         identifier = cdict.get("log_reporter", "reqmgr2")
         self.logdb = LogDB(centralurl, identifier)
 
+        # local team cache which will request data from wmstats
+        self.wmstatsurl = config.reqmgr.wmstats_url
+        if  not self.wmstatsurl:
+            raise Exception('ReqMgr2 configuration file does not provide wmstats url')
+        self.team_update_interval = 3600 # interval we update team cache
+        if  hasattr(config.reqmgr, 'team_update_interval'):
+            self.team_update_interval = int(config.reqmgr.team_update_interval)
```

Comment:
and these 3 lines could be replaced by:
`self.team_update_interval = cdict.get('team_update_interval', 3600)`